docs/docsite/rst/community/development_process.rst

```diff
@@ -82,6 +82,48 @@ pull request to backport the change to a previous stable branch.
     but it can be helpful, especially when making multiple backport PRs for
     multiple stable branches.
 
+.. note::
```

Comment:
This is long enough that I think I'd reformat it as a higher level section.  Maybe 
" Backport Pull Request Process (with vanilla git)" and "Backport Pull Request Process (with helper script)"

Ask acozine for input.

Reply:
:+1:

Reply:
I like adding cherry-picker as an option, but I'd prefer to keep the documentation of how to use it on their docs site, so we don't have to maintain it. I would replace lines 87-126 with a much shorter bit of text:
`If you prefer, you can use CPython's cherry-picker tool to backport commits from devel to stable branches in Ansible. Take a look at the cherry-picker documentation for details on installing, configuring, and using it.`

Reply:
(with a link added for the cherry-picker documentation)

Reply:
It's okay for me, but I already know how to use it, while for newcomers it might be a bit confusing since upstream doc still mainly focuses on CPython now and has additional irrelevant information, like setting up config (which should just exist in ansible repo w/o user having to worry about it).

Reply:
done